docs/concepts/components.md

```diff
@@ -1,13 +1,70 @@
-# Lokomotive Components
+# Lokomotive component
 
-`lokoctl` can be used to manage Lokomotive *components*. A component is a k8s workload which adds
-functionality to a Lokomotive cluster. Without components, a Lokomotive cluster is just a barebone
-k8s cluster. In some cases (e.g. bare metal environments), a Lokomotive cluster may even be
-unusable without certain components.
+`lokoctl` is a command line interface for Lokomotive and its *components*. A component is a
+Kubernetes workload which adds functionality to a Lokomotive cluster. Without components, a
+Lokomotive cluster is just a barebone kubernetes cluster. In some cases (e.g. bare metal
+environments), a Lokomotive cluster may even be unusable without certain components.
 
-Components can take care of tasks such as load balancing, monitoring, authentication, storage and
+Components take care of tasks such as load balancing, monitoring, authentication, storage and
 others.
 
+## Categories of Lokomotive components
+
+### User Authentication
+
+Lokomotive provides the [Dex](../component-reference/components/dex.md) and
+[Gangway](../component-reference/components/gangway.md) components for user authentication via
+OpenID Connect (OIDC). With these components, you can securely manage access to the Kubernetes
+cluster and resources.
+
+Lokomotive also provides the [cert-manager](../component-reference/components/cert-manager.md)
+component for automating the management and issuance of TLS certificates from various issuing
+sources.
+
+### Monitoring/Metrics
+
+Lokomotive provides a
+[prometheus-operator](../component-reference/components/prometheus-operator.md) component that
+creates, configures and manages [Prometheus](https://prometheus.io/) atop Kubernetes.
+
+Lokomotive also provides a [metrics-server](../component-reference/components/metrics-server.md)
+component responsible for collecting resource metrics from kubelets and exposing them in the
+Kubernetes API server through the [metrics API](https://github.com/kubernetes/metrics).
+
+### Storage
+
+Lokomotive provides the [openebs-operator](../component-reference/components/openebs-operator.md) and
+[openebs-storage-class](../component-reference/components/openebs-storage-class.md) components to
+use OpenEBS as the block storage solution for the cluster.
+
+Lokomotive also provides the [rook](../component-reference/components/rook.md) and
+[rook-ceph](../component-reference/components/rook-ceph.md) components for using Rook as the storage
+solution for Lokomotive cluster.
+
+### Load Balancing/Ingress (for Packet platform only)
+
+Lokomotive provides the [MetalLB](../component-reference/components/metallb.md) component for load
+balancing in bare metal environments and [Contour](../component-reference/components/contour.md)
```

Comment:
Saying bare metal environments may sound confusing here, as we also have bare-metal platform. Maybe we can re-phrase it to "in environments without programmatic load balancers?" or something?